tests/integration/kill.bats

```diff
@@ -53,32 +53,74 @@ function teardown() {
 	fi
 
 	runc run -d --console-socket "$CONSOLE_SOCKET" test_busybox
+	# shellcheck disable=SC2031
 	[ "$status" -eq 0 ]
+	cgpath=$(get_cgroup_path "pids")
+	init_pid=$(cat "$cgpath"/cgroup.procs)
 
 	# Start a few more processes.
 	for _ in 1 2 3 4 5; do
 		__runc exec -d test_busybox sleep 1h
-		[ "$status" -eq 0 ]
```

Comment:
There was a bug in the original test case code -- since we're not using bats' `run`, there is no `$status` to check (essentially, this checks the status from the last `run` call which is `runc run ...` above).

Reply:
If needed, I can separate this (and other fixes to the test case) to a separate commit, for clarity.

Reply:
I think it's okay to keep it in the same commit. It's a no-op change anyway (the deleted check would never fail anyway for the reason you described).